HotelWebsite/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <title>HOTEL WEBSITE</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div class="container">
+                <h1 class="logo">HBT</h1>
+                <ul>
+                    <li><a class="current" href="index.html"> HOME</a></li>
+                    <li><a href="about.html">ABOUT </a></li>
+                    <li><a href="contact.html">CONTACT</a></li>
+                </ul>
+            </div>
+        </nav>
+        <div id="showcase">
+            <div class="container">
+                <div class="showcase-content">
+                    <h1> <span class="primary-heading">Enjoy</span> Your Stay</h1>
+                    <p class="lead">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Enim quos laborum eum,
+                        impedit neque ad?
+                    </p>
+                    <a class="btn" href="about.html">About Our Hotel</a>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section id="home-info" class="bg-dark">
+        <div class="info-img">
+
+        </div>
+        <div class="info-content">
+            <h2><span class="primary-heading">The History</span> Of Hotel</h2>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+                Ducimus dicta doloremque fuga rerum cupiditate saepe nesciunt
+                quis obcaecati tempora accusantium, sit labore magni sunt! Quos
+                consequatur sunt rerum dicta adipisci necessitatibus
+                maxime hic, consequatur sunt rerum dicta adipisci necessitatibus
+                maxime hic,consequatur sunt rerum dicta adipisci necessitatibus
+                maxime hic,!</p>
+
+            <a href=" about.html" class="btn btn-light">Read More</a>
+        </div>
+    </section>
+    <div class="clr"></div>
+    <section id="features">
```

Comment:
Try to use classes most of the time, use ids only when unique identification is required